contrib/package_plugin.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python
```

Comment:
This file should be executable:
```bash
chmod +x contrib/package_plugin.py
```

Reply:
I don't use linux, so don't understand what this means, unfortunately.